RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release notes
 
+## New in git-machete 3.7.3
+
+- fixed: *args and **kwargs typing
```

Comment:
Nit: since this is formatted as markdown, pls `\`-escape the asterisks

Reply:
Oh! actually, since 3.7.2 has NOT been released yet, and this change is another bugfix (not a feature that would change CLI), we can stay within **3.7.2**. No need to bump version; I'd actually even skip this entry from changelog b/c it's very low-level/not particularly interesting for the users

Reply:
See https://github.com/VirtusLab/git-machete/blob/master/CONTRIBUTING.md#versioning and the surrounding paragraphs btw